components/forms/form-builder/FormView.tsx

```diff
@@ -106,7 +106,8 @@ export const FormView = ({
       isArcp,
       programmeMembershipId,
       localOfficeName,
-      programmeSpecialty: processedFormData.linkedProgramme?.programmeName
+      programmeSpecialty: processedFormData.linkedProgramme?.programmeName,
```

Comment:
Ah, now that looks like quite a fishy assignment to me (though I could have saved myself a lot of hassle by just using programmeSpecialty in the PDF, and assumed it was ok!)